src/main/java/domain/Lottos.java

```diff
@@ -33,4 +34,13 @@ public Stream stream(){
     public int size() {
         return lottos.size();
     }
+
+    public Lottos addAll(List<Lotto> lottos) {
+        List<Lotto> addedLottos = new ArrayList<>();
+
+        this.lottos.stream().forEach(addedLottos::add);
+        lottos.stream().forEach(addedLottos::add);
```

Comment:
```suggestion
       addedLottos.addAll(this.lottos);
       addedLottos.addAll(lottos);
```

addAll 를 사용해보면 어떨까요? 😄